ParisBNF/abb/BNFabb156.xml

```diff
@@ -0,0 +1,627 @@
+            <msDesc xml:id="ms">
+              <msIdentifier>
+                  <repository ref="INS0303BNF"/>
+                  <collection>Manuscrits orientaux</collection>
+                  <collection>Fonds éthiopien</collection>
+                  <collection>d'Abbadie</collection>
+                 <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b100873516">BnF Éthiopien d'Abbadie 156</idno>
+                    </msIdentifier>
+               <msContents>
+                  <summary>
+                  </summary>
+                  
+                  <msItem xml:id="ms_i1">
+                     <locus from="3ra" to="19ra"/>                     
+                     <title type="complete" ref="LIT6824CommentariesOldTest"><foreign xml:lang="gez">Tǝrgʷāme</foreign>-commentaries on the books of the Octateuch</title>
+                                      
+                     <msItem xml:id="ms_i1.1">
+                              <locus from="3ra" to="4ra"/>
+                        <title type="complete" ref="LIT6824CommentariesOldTest#Introduction">Introduction</title>
+                              <textLang mainLang="gez"/>
+                        <incipit xml:lang="gez"><hi rend="rubric">ስብሐት፡ ለእግዚአብሔር፡ መብርሄ፡ ልቡና፡ ንጽሕፍ፡</hi> ትርጓሜ፡ ኦሪት፨ 
+                        ኦሪት፡ ብሂል፡ ፍትሕ፡ ወሥርዓት፡ ወሕግ<hi rend="rubric">ሂ፡ ይትበሀል። ወቦ፡ እለ፡ እቤሉ፡ አእምሮ፡ ዘይፈልጥ፡</hi>ሠናየ፡ እምእኩይ። 
+                        ወቦ፡ እለ፡ ይብሉ፡ መጽሔት፡ ዘይሬእዩ፡ ባቲ፡ ግብረ፡ እግዚአብሔር። እስመ፡ በሌሊት፡ እሁድ፡ ፈጠረ፡ እግዚአብሔር፡ ፯ፍጥረታተ፡ በአርምሞ። 
+                        ተፈጽመ፡ ፯ሥርዋት፡ ዘውእቶሙ፡ መላእክት፡ ሰማይ፡ 
+                        </incipit>
+                     </msItem>
+
+                              <msItem xml:id="ms_i1.2">
+                                <locus from="4ra" to="9ra"/>
+                                 <title type="complete" ref="LIT4117Interpretation">Interpretation and symbolic commentary on <ref type="work" corresp="LIT1546Genesi">Genesis</ref></title>
+                                 <textLang mainLang="gez"/>
+                                 <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ እግዚአብሔር፡</hi> መሐሪ፡ ወመስተሣህል፡ እወጥን፡ ጽሒፈ፡ መጽሐፍ፡ ዘይሰምይ፡ 
+                                    መምህር፨ ዘአስተጋብእዎ፡ ማእምራነ፡ መጻሕፍት፨ ወረባሁሰ፡ ለዝመጽሐፍ፡ ይሜህር፡ ወይተረጕሞ፡ ቃላተ፡ ወነገራተ፡ 
+                                    ዘኦሪት፡ እንተ፡ የዓፅብ፡ ወይስአን፨ ዘይትሀባእ፡ ወዘይሤውር፡ ለዘ፡ ኢየአምር፡ ብእሲ፨ ወትርጓሜሁሰ፡ በኀበ፡ ይነግር፡ 
+                                    ፩ቃለ፡ በ፫ጾታ፡ ወበበ፪፡ ስብሐት፡ ለእግዚአብሔር፡ በአፈ፡ ኵሉ፡ ፍጥረት፡ <gap reason="ellipsis" resp="DN"/> <hi rend="rubric">ኦሪት፡ ዘልደት፡</hi>
+                                    ኦሪት፡ ብሂል፡ ሕግ፡ ዘልደት፡ ብሂል፡ ዘፍጠት፨ 
+                                 </incipit>
+                              </msItem>
+
+                                 <msItem xml:id="ms_i1.3">
+                                    <locus from="9ra" to="12vb"/>
+                                    <title type="complete"  ref="LIT6824CommentariesOldTest#Exodus">Commentaries on <ref type="work" corresp="LIT1367Exodus">Exodus</ref>
```

Comment:
the `title `is converted to a link, and there can be no link inside of a link, so please no mark up inside of title